src/main/java/core/basesyntax/service/WritingToNewFile.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
```

Comment:
The interface name 'WritingToNewFile' does not follow Java naming conventions. Interface names should be adjectives and often end with '-able' or '-ible'. Consider renaming it to something like 'FileWritable' to better convey its intent.